cli/util.zig

```diff
@@ -320,6 +322,6 @@ const UTF8ConsoleOutput = struct {
     }
 
     fn deinit(self: UTF8ConsoleOutput) void {
-            _ = std.os.windows.kernel32.SetConsoleOutputCP(self.original);
+        _ = std.os.windows.kernel32.SetConsoleOutputCP(self.original);
```

Comment:
my auto-formatter kept doing this so I stopped fighting it

Reply:
@erg No problem - I use zig fmt, I think this code came from somebody else - you can always feel free to apply zig fmt anywhere in Jetzig code.